### ISSUE TITLE
Crash when undoing in graph with Revit nodes in Sandbox

### ISSUE THREAD

**Issue by radumg** [DynamoDS/Dynamo#9910]
## Dynamo version

2.0.2

## Operating system

Windows 10

## What did you do? 

Open graph that has Revit nodes in Dynamo Sandbox.
Move group on canvas
Undo move -> crash

## What did you expect to see?

No crash.

## What did you see instead?

Crash.
Happens only on this graph, trying to move the `get room boundaries` & `combine intersecting...` groups, then undo.

![image](https://user-images.githubusercontent.com/11439624/63049541-d6ef6900-bed0-11e9-8d68-5be423ba0e53.png)

## Stack trace

```
Object reference not set to an instance of an object.

   at Dynamo.Graph.Nodes.CodeBlockNodeModel.LoadAndCreateConnectors(OrderedDictionary inportConnections, OrderedDictionary outportConnections, SaveContext context)
   at Dynamo.Graph.Nodes.CodeBlockNodeModel.DeserializeCore(XmlElement nodeElement, SaveContext context)
   at Dynamo.Graph.Workspaces.WorkspaceModel.ReloadModel(XmlElement modelData)
   at Dynamo.Core.UndoRedoRecorder.UndoActionGroup(XmlElement actionGroup)
   at Dynamo.Graph.Workspaces.WorkspaceModel.Undo()
   at Dynamo.Models.DynamoModel.ExecuteCommand(RecordableCommand command)
   at System.Windows.Input.CommandManager.TranslateInput(IInputElement targetElement, InputEventArgs inputEventArgs)
   at System.Windows.UIElement.OnKeyDownThunk(Object sender, KeyEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndKeyboardInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawKeyboardActions actions, Int32 scanCode, Boolean isExtendedKey, Boolean isSystemKey, Int32 virtualKey)
   at System.Windows.Interop.HwndKeyboardInputProvider.ProcessKeyAction(MSG& msg, Boolean& handled)
   at System.Windows.Interop.HwndSource.CriticalTranslateAccelerator(MSG& msg, ModifierKeys modifiers)
   at System.Windows.Interop.HwndSource.OnPreprocessMessage(Object param)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at System.Windows.Threading.Dispatcher.Invoke(DispatcherPriority priority, Delegate method, Object arg)
   at System.Windows.Interop.HwndSource.OnPreprocessMessageThunk(MSG& msg, Boolean& handled)
   at System.Windows.Interop.ThreadMessageEventHandler.Invoke(MSG& msg, Boolean& handled)
   at System.Windows.Interop.ComponentDispatcherThread.RaiseThreadMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at DynamoSandbox.DynamoCoreSetup.RunApplication(Application app)
```

**Comment by radumg** [comment]
Note : the only thing being undone is moving the group on canvas.

**Comment by mjkkirschner** [comment]
hey @radumg can you please attach the graph that you hit this crash with? It will help us reproduce it.

**Comment by mjkkirschner** [comment]
tracked with https://jira.autodesk.com/browse/DYN-2286

**Comment by QilongTang** [comment]
@radumg While our team decide to tackle this one, do you still have the graph around?

**Comment by radumg** [comment]
hey @QilongTang & @mjkkirschner , sorry for the delay, I'll try and track down what I was working on at that time.

**Comment by radumg** [comment]
it should be this graph here : 

https://github.com/DynamoDS/RefineryToolkits/blob/master/samples/SpacePlanning/dt_GenerativeToolkit_Visibility_LinesTest(RVT).dyn

**Comment by radumg** [comment]
Uploaded here too, just in case
[dt_GenerativeToolkit_Visibility_LinesTest(RVT).zip](https://github.com/DynamoDS/Dynamo/files/3905046/dt_GenerativeToolkit_Visibility_LinesTest.RVT.zip)

**Comment by avidit** [comment]
fixed in 2.6
